notebooks/image_models/solutions/vertex_tuning_image_classification.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    BUCKET = PROJECT + "-flowers"
</pre><p>Can we just use the same bucket that we usually use <code>project_id</code> and then create a folder <code>flowers</code>  under this project to keep it uniform with other labs? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>